input/fsh/Mappings/MappingBodyMeasures.fsh

```diff
@@ -0,0 +1,17 @@
+Mapping: BodyHeightToICHOM
```

Comment:
Same for example - so we have one file that describes the profile, mapping for it, and an example, all inline. It would make organisation easier, do you agree?